lib/resources/script.js

```diff
@@ -15,54 +14,50 @@ function initSideNav() {
   }
 
   if (overlay) {
-    overlay.addEventListener('click', function(e) {
-      toggleBoth();
-    });
+    overlay.addEventListener('click', toggleBoth);
   }
 
   if (leftNavToggle) {
-    leftNavToggle.addEventListener('click', function(e) {
-      toggleBoth();
-    });
+    leftNavToggle.addEventListener('click', toggleBoth);
   }
 }
 
 function saveLeftScroll() {
-  var leftSidebar = document.getElementById('dartdoc-sidebar-left');
-  sessionStorage.setItem('dartdoc-sidebar-left-scrollt' + window.location.pathname, leftSidebar.scrollTop);
-  sessionStorage.setItem('dartdoc-sidebar-left-scrolll' + window.location.pathname, leftSidebar.scrollLeft);
+  const leftSidebar = document.getElementById('dartdoc-sidebar-left');
+  sessionStorage.setItem('dartdoc-sidebar-left-scrollt' + window.location.pathname, leftSidebar.scrollTop.toString());
+  sessionStorage.setItem('dartdoc-sidebar-left-scrolll' + window.location.pathname, leftSidebar.scrollLeft.toString());
 }
 
 function saveMainContentScroll() {
-  var mainContent = document.getElementById('dartdoc-main-content');
-  sessionStorage.setItem('dartdoc-main-content-scrollt' + window.location.pathname, mainContent.scrollTop);
-  sessionStorage.setItem('dartdoc-main-content-scrolll' + window.location.pathname, mainContent.scrollLeft);
+  const mainContent = document.getElementById('dartdoc-main-content');
+  sessionStorage.setItem('dartdoc-main-content-scrollt' + window.location.pathname, mainContent.scrollTop.toString());
+  sessionStorage.setItem('dartdoc-main-content-scrolll' + window.location.pathname, mainContent.scrollLeft.toString());
 }
 
 function saveRightScroll() {
-  var rightSidebar = document.getElementById('dartdoc-sidebar-right');
-  sessionStorage.setItem('dartdoc-sidebar-right-scrollt' + window.location.pathname, rightSidebar.scrollTop);
-  sessionStorage.setItem('dartdoc-sidebar-right-scrolll' + window.location.pathname, rightSidebar.scrollLeft);
+  const rightSidebar = document.getElementById('dartdoc-sidebar-right');
+  sessionStorage.setItem('dartdoc-sidebar-right-scrollt' + window.location.pathname, rightSidebar.scrollTop.toString());
```

Comment:
JS has a lot of automatic type conversions, particularly around numbers and strings. These changes including the `parseFloat`,  probably weren't necessary but I thought it would be good to be explicit.